olp-cpp-sdk-dataservice-read/src/QueryMetadataJob.h

```diff
@@ -132,6 +131,10 @@ class QueryMetadataJob {
 
       auto download_job = download_job_;
 
+      std::this_thread::sleep_for(std::chrono::milliseconds(1000));
```

Comment:
Is this a leftover from testing? Why should we wait for 1s here?

Reply:
Yes, will remove